packages/mobile/src/recipients/recipient.ts

```diff
@@ -213,8 +214,8 @@ function fuzzysortToRecipients(
 }
 
 function nameCompare(a: FuzzyRecipient, b: FuzzyRecipient) {
-  const nameA = a.displayName.toUpperCase()
-  const nameB = b.displayName.toUpperCase()
+  const nameA = a.displayName ? a.displayName.toUpperCase() : ''
```

Comment:
small nit: this could be written as `a.displayName?.toUpperCase() ?? ''` which looks a bit nicer imo, but I guess that's subjective :P